client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -195,6 +195,10 @@ UndocumentedSite.prototype.postCounts = function( options, callback ) {
 	return this.wpcom.req.get( '/sites/' + this._id + '/post-counts/' + type, query, callback );
 };
 
+UndocumentedSite.prototype.postTypeTaxonomies = function( postType, callback ) {
```

Comment:
we have to encourage to work directly in wpcom.js instead of add more code into undocumented. I'm aware it's get a bit annoying but as far as see it's the only one decent way to do it.
We can write a github doc page in wpcom.js talking about wpcom.js workflow.